### PR TITLE
wikia-stats-timeout | Changed discussions stats request timeout

### PR DESCRIPTION
**Pull request by Bloodmoon24**
@kvas-damian @bkoval please CR

### CODE REVIEW

File: includes/wikia/services/WikiDetailsService.class.php [hunk 1]
```diff
@@ -282,7 +282,7 @@ private function getDiscussionStats( $id ) {
 		global $wgConsulServiceTag, $wgConsulUrl;
 
 		$consulUrl = ( new Wikia\Service\Gateway\ConsulUrlProvider( $wgConsulUrl, $wgConsulServiceTag ))->getUrl( 'discussion' );
-		$response = Http::get( "http://$consulUrl/$id/forums/$id?limit=1", 'default', array( 'noProxy' => true ));
+		$response = Http::get( "http://$consulUrl/$id/forums/$id?limit=1", 3, array( 'noProxy' => true ));
```
Comment (dianafa) [inline includes/wikia/services/WikiDetailsService.class.php]: can we move this magic number to a variable indicating what it does/is for?

Comment (bkoval) [inline includes/wikia/services/WikiDetailsService.class.php]: +1

File: includes/wikia/services/WikiDetailsService.class.php [hunk 2]
```diff
@@ -281,8 +281,10 @@ protected function cacheWikiData( $wikiInfo, $method = null ) {
 	private function getDiscussionStats( $id ) {
 		global $wgConsulServiceTag, $wgConsulUrl;
 
+		$discussionsRequestTimeout = 3;
```
Comment (kvas-damian) [inline includes/wikia/services/WikiDetailsService.class.php]: What do you think about using constant?

Comment (dianafa) [inline includes/wikia/services/WikiDetailsService.class.php]: +1

**Pull request closed without merging**